src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -778,7 +783,7 @@ class EthService(
 
       val tx = Transaction(0, req.tx.gasPrice, gasLimit, toAddress, req.tx.value, req.tx.data)
       val fakeSignature = ECDSASignature(0, 0, 0.toByte)
-      SignedTransaction(tx, fakeSignature, fromAddress)
+      SignedTransactionWithSender(SignedTransaction(tx, fakeSignature), fromAddress)
```

Comment:
maybe you should consider adding object for `SignedTransactionWithSender ` with apply method something like that:
```
def apply(transaction: Transaction, signature: ECDSASignature, sender: Address): SignedTransactionWithSender = {
    SignedTransactionWithSender(SignedTransaction(transaction, signature), sender)
  }
```
to get rid of always adding `SignedTransaction `